gb-lcd/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "gb-lcd"
+version = "0.1.0"
+authors = ["Guilhem Smith <gsmith@student.42.fr>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
the dependencies are listed 2 time here at `gb-lcd/Cargo.toml` and at `Cargo.toml`, what's the reason ?

Reply:
Both the main crate and `gb-lcd` need to call functions from `sdl2` and `egui`. Maybe we can import these dependencies only in `gb-lcd` and export them so the main crate can still have access to them ?

Reply:
I understand, I think it's best to keep  the duplicated import because the main crate & `gb-lcd` use them